tests/infra/ccf.py

```diff
@@ -527,8 +561,8 @@ def stop_all_nodes(self):
     def get_members(self):
         return self.members
 
-    def get_running_nodes(self):
```

Comment:
No need for this function anymore. We only care whether nodes have joined the network, not that they are running.